packages/electron-builder/src/publish.ts

```diff
@@ -31,7 +32,7 @@ export function configurePublishCommand(yargs: yargs.Argv): yargs.Argv {
       description: "The app/build version used when searching for an upload release (used by some Publishers)",
     })
     .option("config", {
-      alias: ["c"],
+      alias: ["c", "configurationFilePath"],
```

Comment:
To ensure backward compatibility of the 'publish' function, I added aliases to the command options instead of changing the function parameter names.